### ISSUE TITLE
use alt attribute on images

### ISSUE THREAD

**Issue by rimmesbe** [travisfrazier/teacozy#2]
Try to get in the habit of describing your images with the alt attribute. This description will show up incase your image fails to load, it is read by screen readers, and it just makes your code look more professional.

https://github.com/travisfrazier/teacozy/blob/master/index.html#L39
